CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
-- Nothing.
+- [#7](https://github.com/mezzio/mezzio-fastroute/pull/7) Add PHP 8.0 support
```

Comment:
this can be reverted, should be handled by automatic release as far as I remember

Reply:
In my last Merge Requests i added this manually or it was added by the maintainer.
I didn't find anything in the https://github.com/laminas/automatic-releases project.

if this could break something i will revert my last commit fb01ef397f95186896ce0ec74f604cd0d16fe196 which only sets the Changelog.

For me it would be good to know if it's handled automatically or not.

Reply:
@marcelthole
A bit hidden in the list of steps of the workflow:

> 3. create a changelog by looking at the milestone description and associated issues and pull requests

https://github.com/laminas/automatic-releases#usage

This means the title of the issue report / pull request is used to create the changelog.

Reply:
Awesome. Thanks. I reverted the last commit :)